app/views/smart_proxies/show.html.erb

```diff
@@ -52,6 +55,9 @@
         </div>
       </div>
       <div class="col-md-6">
+      <% sorted_pagelets_at(:katello_proxy_details).each do |pagelet| %>
+        <%= render pagelet.partial, {:smart_proxy => @smart_proxy } %>
+      <% end %>
```

Comment:
We should avoid naming plugins in extension point names, discovery can add something here as well. I suggest `details_contents` or similar.